sdk/core/azure-core-amqp/test/ut_uamqp/uamqp_value_tests.cpp

```diff
@@ -215,4 +220,26 @@ TEST_F(TestValue, SimpleCreate)
     GTEST_LOG_(INFO) << "Handle Type: " << amqpvalue_get_type(handle.get())
                      << " Value: " << static_cast<AMQP_VALUE>(handle.get());
   }
+
+  {
+    AmqpValue descriptor(235ll);
+    AmqpValue descriptorValue("Value");
+    _detail::UniqueAmqpValueHandle handle{amqpvalue_create_described(
+        _detail::AmqpValueFactory::ToUamqp(descriptor), _detail::AmqpValueFactory::ToUamqp(descriptorValue))};
+
+    AmqpValue value{_detail::AmqpValueFactory::FromUamqp(handle)};
+    GTEST_LOG_(INFO) << "Handle Type: "
+                     << amqpvalue_get_type(amqpvalue_get_inplace_descriptor(handle.get()))
+                     << " Value: " << static_cast<AMQP_VALUE>(handle.get());
+  }
+  {
+    AmqpValue descriptor(235ll);
+    _detail::UniqueAmqpValueHandle handle{
+        amqpvalue_create_composite(_detail::AmqpValueFactory::ToUamqp(descriptor), 21)};
+
+    AmqpValue value{_detail::AmqpValueFactory::FromUamqp(handle)};
+    GTEST_LOG_(INFO) << "Handle Type: "
+                     << amqpvalue_get_type(amqpvalue_get_inplace_descriptor(handle.get()))
+                     << " Value: " << static_cast<AMQP_VALUE>(handle.get());
+  }
```

Comment:
I am not seeing `EXPECT_EQs`, or something like that.

Reply:
With this, we may be able to see higher coverage number, but it may not mean what we think it means.

Reply:
Those checks are in the mainline tests.

This module exists specifically to test interoperability with uAMQP constructs in ways that cannot be done in the mainline tests. So there are already tests that verify that AmqpDescribed types work correctly and interoperate with uAMQP.

These tests are literally about increasing code coverage of the ostream inserters used to dump the underlying AMQPVALUE and AMQP_TYPE types.